tests/tests/test_timetable.py

```diff
@@ -166,6 +166,69 @@ def test_conflicts(
     )
 
 
+@pytest.mark.parametrize(
+    ["paced_train_interval", "expected_conflict_types"],
+    [
+        ("PT15M", set()),  # Every half-hour, no conflict between each occurrences
+        (
+            "PT1M",
+            set({"Spacing"}),
+        ),  # Every second, all occurrences will fight for space
+    ],
+)
+def test_paced_train_conflicts(
+    small_infra: Infra,
+    timetable_id: int,
+    fast_rolling_stock: int,
+    paced_train_interval: str,
+    expected_conflict_types: set[str],
+):
+    requests.post(f"{EDITOAST_URL}infra/{small_infra.id}/load").raise_for_status()
+    paced_train_payload = {
+        "comfort": "STANDARD",
+        "constraint_distribution": "STANDARD",
+        "initial_speed": 0,
+        "labels": [],
+        "options": {"use_electrical_profiles": False},
+        "path": [
+            {"id": "start", "track": "TC1", "offset": 185000},
+            {"id": "end", "track": "TD0", "offset": 24820000},
+        ],
+        "power_restrictions": [],
+        "rolling_stock_name": "fast_rolling_stock",
+        "schedule": [
+            {
+                "at": "start",
+            },
+            {
+                "at": "end",
+            },
+        ],
+        "speed_limit_tag": "MA100",
+        "start_time": "2024-05-22T08:00:00.000Z",
+        "train_name": "paced train",
+        "paced": {"time_window": "PT1H", "interval": paced_train_interval},
+        "exceptions": [],
+    }
+
+    stopping_paced_train_response = requests.post(
```

Comment:
Probably a left over from copy-paste:
```suggestion
    paced_train_response = requests.post(
```

Reply:
Done in [1894a19](https://github.com/OpenRailAssociation/osrd/compare/6ec2e2776e0cb44b86e035d89ce32697642a57ed..1894a19de130e593ad6441fd361da25130252bb8).